core/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -469,6 +479,25 @@ public final GetResult get(Get get) throws EngineException {
     public abstract GetResult get(Get get, Function<String, Searcher> searcherFactory) throws EngineException;
 
     /**
+     * Return a tuple representing the sequence ID for the given {@code Get}
+     * operation. The first value in the tuple is the sequence number, the
+     * second is the primary term.
+     */
+    public final Tuple<Long, Long> getSequenceID(Get get) throws EngineException {
```

Comment:
As long as I can make it not an inner class :D 

I'll update this to use an actual class